bin/bst-test.ts

```diff
@@ -1,17 +1,26 @@
 #!/usr/bin/env node
 import * as program from "commander";
 import {Global} from "../lib/core/global";
+import {BstStatistics, BstCommand} from "../lib/statistics/bst-statistics";
 
 const skillTesting = require("skill-testing-ml");
 
 program.version(Global.version());
 
-program
-    .usage("[test-pattern-regex]")
-    .description("Runs unit-tests for a skill - automatically searches for YML test files and runs them")
-    .parse(process.argv);
+Global.initializeCLI().then(() => {
```

Comment:
Can we add an optional parameter to this, which does not generate an ID if there is not already one?

This is because we are commonly going to use this command with CI/CD tools, where they will constantly be re-creating this. We do not want to continuously generate new IDs.

There should be a new flag, bootstrap, that gets passed to this method:
https://github.com/bespoken/bst/blob/master/lib/client/bst-config.ts#L23

If it is set to false, then it will not create a new ID (i.e., it will not call the bootstrapIfNeeded method).

By default, it should be set to true, as that is typically what we want.